components/input/index.jsx

```diff
@@ -122,6 +122,3 @@ Input.defaultProps = {
   prefixCls: 'ant-input',
   type: 'text',
 };
-
-module.exports = Input;
-module.exports.Group = Group;
```

Comment:
That will be incompatible.

```js
import { Input } from antd;
const Group = Input.Group;  // broken here
```

Maybe this will be better:

```js
...

Input.Group = Group;
export default Input;
```